src/pages/FlowRun/Restart-Dialog.vue

```diff
@@ -151,40 +157,19 @@ export default {
       Restart from failed?
     </v-card-title>
 
-    <v-card-text v-if="hasResultHandler">
+    <v-card-text>
       Click on confirm to restart
       <span class="font-weight-bold">{{ flowRun.name }}</span> from its failed
```

Comment:
```suggestion
      <span class="font-weight-bold">{{ flowRun.name }}</span> 
```

Reply:
Since we can now restart without a failed task run, we should maybe take this bit out.